spec/views/events/edit.html.erb_spec.rb

```diff
@@ -6,23 +6,23 @@
     @user = FactoryBot.create :user
     sign_in @user
     @event = assign(:event, Event.create!(
-      :name => "MyString",
-      :description => "MyText",
-      :owner => FactoryBot.build(:user),
-      :game_mode => "MyString",
-      :max_teams => 20,
-      :player_type => Event.player_types[Event.player_types.keys.sample],
-      :discipline => "MyString",
-      :deadline => Date.new(2017,11,16),
-      :startdate => Date.new(2017,12,01),
-      :enddate => Date.new(2017,12,05)
+                              :name => "MyString",
```

Comment:
While we're at it: Can we turn on [`Style/HashSyntax`](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/hash_syntax.rb), which forces the new hash syntax on symbol key only hashes?

Reply:
Yes, you're right. I'll do it.

Reply:
@Pottiman it looks to me like `ruby19_no_mixed_keys` from http://www.rubydoc.info/gems/rubocop/0.45.0/RuboCop/Cop/Style/HashSyntax would fit us best. Did you happen to see another option which discourages use of symbols as keys where hash rockets are allowed again?

Reply:
I think the default (`ruby19`) allows rockets in non symbol key hashes, eg
```ruby
a = {
  :a => 1,
  'b' => 2
}
```
is legal.
Do you want to explicitly prohibit this?:
```ruby
a = {
  a: 1,
  'b' => 2
}
```

Reply:
As discussed, let's not do this.